l1-contracts/contracts/bridge/asset-router/L2AssetRouter.sol

```diff
@@ -153,12 +159,13 @@ contract L2AssetRouter is AssetRouterBase, IL2AssetRouter {
         });
 
         bytes memory message;
-        if (L2_LEGACY_SHARED_BRIDGE == address(0)) {
+        if (_alwaysNewMessageFormat || L2_LEGACY_SHARED_BRIDGE == address(0)) {
             message = _getAssetRouterWithdrawMessage(_assetId, _l1bridgeMintData);
             // slither-disable-next-line unused-return
             L2ContractHelper.sendMessageToL1(message);
         } else {
             address l1Token = IL2NativeTokenVault(L2_NATIVE_TOKEN_VAULT_ADDR).tokenAddress(_assetId);
+            require(l1Token != address(0), "Unsupported asset Id by NTV");
```

Comment:
please use custom errors (I understand we don't use them in all places, but it is better not to add more work for us in the future)

Reply:
Not for the 2 am pr though